### DISCUSSION TITLE
Support wildcard in repositories

### DISCUSSION THREAD

**Discussion by vo55** [renovatebot/renovate#8424] [Request Help] [answered]
It would be great to be able to call renovate on a project level with something like `renovate/renovate <project>/*`.
The expected behaviour would be, that renovate discovers all repositories within that project and executes the dependency update for these.

As far as i know currently it is possible to either call renovate on a repository level or with the autodiscover flag.

**Comment by viceice** [comment, marked as answer]
Checkout [autodiscoverFilter](https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter) 🙃

**Reply by vo55** [reply]
Thank you... 🤦Big facepalm here.
